changelog_entry.yaml

```
@@ -0,0 +1,5 @@
+- bump: minor
+  changes:
+    added:
+    deleted: me_personal_exemptions.py
```

Per https://keepachangelog.com/
```suggestion
    removed:
      - `me_personal_exemptions` variable, instead using the existing num variable in the `me_personal_exemption_deduction` formula.
```